source/leuven/services/openondemand.rst

```diff
@@ -419,14 +419,17 @@ For more general information, please refer to the `official JupyterLab documenta
 RStudio Server
 --------------
 
-This interactive app allows you to run an RStudio session as a compute job.
-You will be running RStudio with R version 4.2.1.
-For more information on how to use RStudio, check out the `RStudio official documentation`_.
+This interactive app allows you to run an RStudio session on the cluster. 
+In the 'Toolchain year and R version' drop-down menu, you can control the R installation
+that will be used for this session (e.g. the `R/4.2.2-foss-2022b` module).
+Additionally, the R-bundle-CRAN and R-bundle-Bioconductor modules can be loaded 
+on top of the base R module to provide easy access to hundreds of preinstalled packages.
 
-The use is very similar to regular RStudio.
-It is recommended to install packages in a folder on your ``$VSC_DATA`` instead of the default location though,
-to avoid clogging your ``$VSC_HOME``.
-You can do this by using the ``lib`` argument for both the ``install.packages`` and the ``library`` function.
+It is also possible to use locally installed R packages with RStudio, see :ref:`R package management<r_package_management_standard_lib>`. 
+While the use of RStudio projects is also available within RStudio, loading such project may affect your module library paths as well as your personal R library path.
+We therefore recommend to only use RStudio projects with `renv <https://rstudio.github.io/renv/articles/renv.html>`__, ensuring a completely isolated library for your project.
+
+For more information on how to use RStudio, check out the `official documentation <https://docs.rstudio.com/>`__.
```

Comment:
https://docs.rstudio.com redirects to https://docs.posit.com and it looks like it would be better to link to a more RStudio-specific page on that website:
```suggestion
For more information on how to use RStudio, check out the `official documentation <https://docs.posit.co/ide/user/>`__.
```